src/itensor.jl

```diff
@@ -582,8 +582,8 @@ function dense(A::ITensor)
 end
 
 """
-    randomITensor([::Type{ElT <: Number} = Float64, ]inds)
-    randomITensor([::Type{ElT <: Number} = Float64, ]inds::Index...)
+    randomITensor([rng=default_rng(), ::Type{ElT <: Number} = Float64, ]inds)
+    randomITensor([rng=default_rng(), ::Type{ElT <: Number} = Float64, ]inds::Index...)
```

Comment:
```suggestion
    randomITensor([rng=Random.default_rng(), ][::Type{ElT <: Number} = Float64, ]inds)
    randomITensor([rng=Random.default_rng(), ][::Type{ElT <: Number} = Float64, ]inds::Index...)
```
to make it clearer that `default_rng()` isn't exported from `Random`, and also make it clearer that `rng` and element type can be specified separately.